lib/content_security_policy/default.rb

```diff
@@ -60,14 +60,17 @@ def script_assets(
     end
 
     def script_src
-      [
-        "#{base_url}/logs/",
-        "#{base_url}/sidekiq/",
-        "#{base_url}/mini-profiler-resources/",
-        *script_assets,
-      ].tap do |sources|
-        sources << :report_sample if SiteSetting.content_security_policy_collect_reports
-        sources << :unsafe_eval if Rails.env.development? # TODO remove this once we have proper source maps in dev
+      sources = []
+
+      if SiteSetting.content_security_policy_strict_dynamic
+        sources << "'strict-dynamic'"
+      else
+        sources.push(
+          "#{base_url}/logs/",
+          "#{base_url}/sidekiq/",
+          "#{base_url}/mini-profiler-resources/",
+          *script_assets,
+        )
 
         # Support Ember CLI Live reload
         if Rails.env.development?
```

Comment:
Do we still need: 

```
        if Rails.env.development?
          sources << "#{base_url}/ember-cli-live-reload.js"
          sources << "#{base_url}/_lr/"
        end
```

Not a blocker, but it seems to be unnecessary with strict dynamic and it raises a console error locally.

Reply:
We don't need it in the strict-dynamic world. But this code is in the `else` part of the if-statement, so it only applies to the old world.

What console error are you seeing locally? Make sure you've restarted ember-cli since pulling this branch - there have been some changes to the proxy logic.

Reply:
Ah, it's the ember-cli restarting. The error was a CSP violation with `ember-cli-live-reload.js`, but it's gone after restarting the cli. 👍